spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/controller/SkipperController.java

```diff
@@ -63,18 +66,29 @@
 	private SkipperStateMachineService skipperStateMachineService;
 
 	public SkipperController(ReleaseService releaseService, PackageService packageService,
+			PackageMetadataService packageMetadataService,
 			SkipperStateMachineService skipperStateMachineService) {
 		this.releaseService = releaseService;
 		this.packageService = packageService;
+		this.packageMetadataService = packageMetadataService;
 		this.skipperStateMachineService = skipperStateMachineService;
 	}
 
+	// Package commands
+
 	@RequestMapping(path = "/upload", method = RequestMethod.POST)
 	@ResponseStatus(HttpStatus.CREATED)
 	public PackageMetadata upload(@RequestBody UploadRequest uploadRequest) {
 		return this.packageService.upload(uploadRequest);
 	}
 
+	@RequestMapping(path = "/package/{name}", method = RequestMethod.DELETE)
+	@ResponseStatus(HttpStatus.OK)
+	public void packageDelete(@PathVariable("name") String name) {
```

Comment:
While this REST endpoint allows us to delete all the package metadata by the given name, delete by name/version could definitely be more useful. We can optionally allow specifying any valid `local` repo name to search for the package metadata as well.

Reply:
Yes, but we can add additional variations another time